api/templates/design.go

```diff
@@ -657,11 +657,15 @@ var VersionDetail = Type("VersionDetail", func() {
 	Attribute("components", ArrayOf(String), "List of components used in the template version.", func() {
 		Example([]string{"ex-generic-v2", "keboola.snowflake-transformation"})
 	})
+	Attribute("longDescription", String, "Extended description of the template in Markdown format.", func() {
+		MinLength(1)
+		Example("Full workflow to load all user accounts from the Service. With extended explanation ...")
```

Comment:
Jen mini nápad; Tady by se taky mohl použít Markdown, ať je ta podpora vidět na první pohled.

Reply:
jako v tom swagger ui, jo? Aha, zkusím

Reply:
Nevím jestli to Swagger vykreslí (a asi je to i jedno), ale jen aby to bylo v tomhle example kódu.

Reply:
ok, je tam